test/parallel/test-net-listen-multiple-calls.js

```diff
@@ -0,0 +1,17 @@
+const assert = require('assert');
+const net = require('net');
+const server = net.createServer();
+server.on('error', () => {
+  process._rawDebug('server had an error');
+});
+assert.throws(function() {
+  server.listen(8080).listen(8081);
```

Comment:
In most of the machines both these ports will be used by other programs. So there is a chance that it fails very often.